user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.templates.js

```diff
@@ -31,7 +31,7 @@ define([
           "<div class='body' role='tree' aria-labelledby='foldersHeader'></div>" +
           "</div>" +
           "<div id='fileBrowserFiles' class='span4 well fileBrowserColumn'>" +
-          "<div class='body'></div>" +
+          "<select class='body' aria-labelledby='filesHeader' multiple size='0' id='MyID'></select>" +
```

Comment:
Could you give a more meaningful identifier? Or, is it forgotten and not needed?